src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import './Button.css';
+
+const Button = ({ name }) => {
+  return <button className='Button'>{name}</button>;
+};
+
```

Comment:
Hey, please use propTypes to validate what is coming here for name is a string

Reply:
Hello. I've done the needful.